src/basic/missing_syscall.h

```diff
@@ -33,39 +33,47 @@ static inline int missing_pivot_root(const char *new_root, const char *put_old)
 
 /* ======================================================================= */
 
-#if !HAVE_MEMFD_CREATE
+#if defined __x86_64__
+#  define fallback_NR_memfd_create 319
```

Comment:
is "fallback" the best name for this?

maybe use `fixed` or `known` or `validated` or so?

Reply:
sure